packages/mesos/buildinfo.json

```diff
@@ -3,8 +3,8 @@
   "single_source" : {
     "kind": "git",
     "git": "https://github.com/mesosphere/mesos",
-    "ref": "8a7340f7ccd06261e17b80a04c3e60a4dd1480e5",
-    "ref_origin" : "dcos-mesos-master-63f116a"
+    "ref": "f7f6c4991337d3667a589927f0f2b58193eeeffe",
+    "ref_origin" : "dcos-mesos-master-78695e4"
```

Comment:
If this is from Apache branch 1.5.x and not master, please rename the branch `dcos-mesos-1.5.x-78695e4` (note that 5 more commits have been added to 1.5.x since this SHA)